### PR TITLE
CANS-683. Add text description next to N/A on specific questions in CANS Assessment Form

### PR DESCRIPTION
**Pull request by denys-davydov**
## JIRA
https://osi-cwds.atlassian.net/browse/CANS-683

Note: depends on https://github.com/ca-cwds/cans-api/pull/169

## Tests

- [x] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/Application/components/Assessment/ItemDescriptionRating.js [hunk 1]
```diff
@@ -7,63 +7,86 @@ import FormControl from '@material-ui/core/FormControl'
...
+  if (!isBooleanRating) return index
+  if (index === 0) return 'No'
+  if (index === 1) return 'Yes'
+  return index
+}
+
+class ItemDescriptionRating extends PureComponent {
+  constructor(props) {
+    super()
+    const ratingDescriptions = props.hasNaOption ? props.ratingDescriptions.slice(0, -1) : props.ratingDescriptions
```
Comment (denys-davydov) [inline app/javascript/Application/components/Assessment/ItemDescriptionRating.js]: Description for N/A value goes as the last element in props.ratingDescriptions

**Comment by denys-davydov** [comment]
Retest this please

**Comment by LeonidMarushevskyi** [comment]
Retest this please

**Comment by LeonidMarushevskyi** [comment]
Retest this please

**Comment by denys-davydov** [comment]
Retest this please

**Comment by denys-davydov** [comment]
Retest this please

**Pull request merged**